src/sorted_vector.jl

```diff
@@ -0,0 +1,52 @@
+"""
+A `SortedVector` behaves like a standard Julia `Vector`, except that its elements are stored in sorted order, with an optional function `by` that determines the sorting order in the same way as `Base.searchsorted`.
```

Comment:
Wrap at 92 chars. Also, better refer to `sort` than to  `searchsorted`. The docstring should also mention explicitly what happens on insert.